neuralhydrology/datasetzoo/camelsus.py

```diff
@@ -209,8 +209,7 @@ def load_camels_us_discharge(data_dir: Path, basin: str, area: int) -> pd.Series
     """
 
     discharge_path = data_dir / 'usgs_streamflow'
-    files = list(discharge_path.glob('**/*_streamflow_qc.txt'))
-    file_path = [f for f in files if f.name[:8] == basin]
+    file_path = list(discharge_path.glob(f'**/{basin}_*_streamflow_qc.txt'))
```

Comment:
```suggestion
    file_path = list(discharge_path.glob(f'**/{basin}_streamflow_qc.txt'))
```
Little copy-paste error I think, but there is one additional underscore here that is not needed.